addons/product/models/product_product.py

```diff
@@ -425,6 +435,221 @@ def _onchange_uom_id(self):
             }
         }
 
+    @api.depends('product_template_attribute_value_ids')
+    def _export_attribute_values(self):
+        for p in self:
+            p.import_attribute_values = ','.join(sorted(
+                f'{v.attribute_line_id.attribute_id.name}:{v.product_attribute_value_id.name}'
+                for v in p.product_template_attribute_value_ids
```

Comment:
Please use decent variable names, `p` should be `product` (or `prod` if too long), and `v` should be... `ptav` ?  Yeah, I know 😄